docs/CONFIG-VARS.md

```diff
@@ -263,23 +263,63 @@ When `storage_type=ha`, [AWS Elastic File System](https://aws.amazon.com/efs/) s
-| postgres_server_name | Name of PostgreSQL server | string | "" | Changing this value trigger resource recreation |
-| postgres_server_version | The version of the PostgreSQL server | string | "11" | Changing this value trigger resource recreation |
-| postgres_instance_type | The VM type for the PostgreSQL Server | string | "db.m5.xlarge" | |
-| postgres_storage_size | Max storage allowed for the PostgreSQL server in MB | number | 50 |  |
-| postgres_backup_retention_days | Backup retention days for the PostgreSQL server | number | 7 | Supported values are between 7 and 35 days. |
-| postgres_storage_encrypted | Encrypt PostgreSQL data at rest | bool | false| |
-| postgres_administrator_login | The Administrator Login for the PostgreSQL Server | string | "pgadmin" | Changing this forces a new resource to be created |
-| postgres_administrator_password | The Password associated with the postgres_administrator_login for the PostgreSQL Server | string | | |
-| postgres_db_name | Name of database to create | string | "SharedServices" | |
-| postgres_multi_az | Specifies if PostgreSQL instance is multi-AZ | bool | false | |
-| postgres_deletion_protection | Protect from accidental resource deletion | bool | false | |
-| postgres_ssl_enforcement_enabled | Enforce SSL on connections to PostgreSQL server instance | bool | true | |
-| postgres_parameters | additional parameters for PostgreSQL server | list of maps | [] | |
-| postgres_options | additional options for PostgreSQL server | list of maps | [] |   |
+| server_version | The version of the PostgreSQL server | string | "11" | Changing this value trigger resource recreation |
+| instance_type | The VM type for the PostgreSQL Server | string | "db.m5.xlarge" | |
+| storage_size | Max storage allowed for the PostgreSQL server in MB | number | 50 |  |
+| backup_retention_days | Backup retention days for the PostgreSQL server | number | 7 | Supported values are between 7 and 35 days. |
+| storage_encrypted | Encrypt PostgreSQL data at rest | bool | false| |
+| administrator_login | The Administrator Login for the PostgreSQL Server | string | "pgadmin" | The admin login name can not be 'admin', must start with a letter, and must be between 1-16 characters in length, and can only contain underscores, letters, and numbers. Changing this forces a new resource to be created |
+| administrator_password | The Password associated with the administrator_login for the PostgreSQL Server | string | "my$up3rS3cretPassw0rd" | The admin passsword must have more than 8 characters, and be composed of any printable characters except the following / ' \" @ characters. |
+| multi_az | Specifies if PostgreSQL instance is multi-AZ | bool | false | |
+| deletion_protection | Protect from accidental resource deletion | bool | false | |
+| ssl_enforcement_enabled | Enforce SSL on connections to PostgreSQL server instance | bool | true | |
+| parameters | additional parameters for PostgreSQL server | list of maps | [] | |
```

Comment:
Can we have links to valid values for parameters and options? This would be similar to the link available for the database_flags option in viya4-iac-gcp.